openapi/connector_mgmt-private.yaml

```diff
@@ -383,6 +387,24 @@ components:
           description: the version of the operator
           type: string
 
+    ConnectorNamespaceStatus:
```

Comment:
can you add some detail about how to use such object ?

i.e:
- I recall we discussed about adding the count of connectors but you said it was not needed
- how do you expect we use the phase/version/conditions ?

Reply:
clarified